src/renderer/components/+catalog/catalog.module.css

```diff
@@ -19,43 +19,45 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-.list :global(.TableRow) {
-  .entityName {
-    position: relative;
-    width: min-content;
-    max-width: 100%;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    padding-right: 24px;
-
-    .pinIcon {
-      position: absolute;
-      right: 0;
-      opacity: 0;
-      transition: none;
-
-      &:hover {
-        /* Drop styles defined for <Icon/> */
-        background-color: transparent;
-        box-shadow: none;
-      }
+.Catalog {}
```

Comment:
A leftover to cleanup.

Reply:
Well, not really. I intentionally left it here and linked to component's top element.
That way it's way can more safe define styles for some inner global class names, e.g.:

```scss
.Catalog {
 :global(.TableRow){ ... }
}
```

Moreover I would prefer to have such component-class name for our each React component with style file.